tests/trainer/resources/disconnected_env/install_kubeflow.py

```
@@ -183,7 +183,31 @@ def install_from_s3():
 
+def install_from_git():
+    """Install kubeflow from git repo (for unreleased versions not yet on Red Hat PyPI)."""
+    git_url = os.environ.get(
+        "KUBEFLOW_GIT_URL",
+        "kubeflow @ git+https://github.com/opendatahub-io/kubeflow-sdk.git"
+    )
+    print(f"Installing kubeflow from git: {git_url}")
+    cmd = [
+        sys.executable, "-m", "pip", "install", "--quiet", "--no-cache-dir", git_url
+    ]
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)